script.js

```diff
@@ -1,53 +1,72 @@
-let findInfoButton = document.getElementById("findInfo")
+let findInfoButton = document.getElementById("findInfo");
 
-let county = document.getElementById("county")
+let county = document.getElementById("county");
 
-findInfoButton.addEventListener('click', logData)
+findInfoButton.addEventListener("click", logData);
 
-const apiUrl = 'https://nominatim.openstreetmap.org/search?format=json&county='
+const apiUrl = "https://nominatim.openstreetmap.org/search?format=json&county=";
 
-function logData(e){
-    e.preventDefault(); 
-    
-    fetch(`${apiUrl}${county.value}`, {
-        method: 'GET'
-    })
-    .then ((resp => resp.json()))
-    .then(data => renderData(data))
-}
-
-let whichCounty = document.getElementById("pick-your-county")
+function logData(e) {
+  e.preventDefault();
 
-function renderData(data){
-    whichCounty.innerText = "Which of these county's is yours?";
+  if (document.getElementById("county-list")) {
+    document.getElementById("county-list").remove();
+  }
 
-    data.forEach(iterateData)
+  fetch(`${apiUrl}${county.value}`, {
+    method: "GET",
+  })
+    .then((resp) => resp.json())
+    .then((data) => {
+      if (data.length) {
+        renderData(data);
+      } else {
+        error();
+      }
```

Comment:
This is a great spot to use some fun ternary operators instead of if/else, if you want:

```suggestion
      data.length ? renderData(data) : error()
```